modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/rebalance/ZoneRebalanceUtil.java

```diff
@@ -425,6 +426,9 @@ private static String zoneInfo(CatalogZoneDescriptor zoneDescriptor) {
     /** Key prefix for switch append assignments. */
     public static final String ASSIGNMENTS_SWITCH_APPEND_PREFIX = "zone.assignments.switch.append.";
 
+    /** Key prefix for assignments chain. */
+    public static final String ASSIGNMENTS_CHAIN_PREFIX = "zone.assignments.chain.";
```

Comment:
Let's remove assignments chain related changes from this PR. Seems like they are not needed on this stage